ui/app/actions.js

```diff
@@ -871,7 +871,8 @@ function pairUpdate (coin) {
 }
 
 function shapeShiftSubview (network) {
-  var pair = 'btc_eth'
+  var pair
+  network === 'classic' ? pair = 'btc_etc' : pair = 'btc_eth'
```

Comment:
I'm not a fan of having these ternary `classic` checks all over the place. ETC isn't the only non-ETH EVM blockchain that wants to be MetaMask compatible (Rootstock is another, for example).

Rather than doing this custom coding by hand, it would be nicer to use a general network config file for interpreting chain data into app behavior.